scripts/ci_install_pyo3_builds.sh

```diff
@@ -8,9 +8,9 @@ install_lib() {
         rm linux.zip && rm -rf x86_64*
 }
 
-install_lib "v1.2.1"
+install_lib "v1.4.0"
```

Comment:
```suggestion
install_lib "v2.1.0"
```
2.1.0 should be the next release version